docker/install_epics.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
```

Comment:
Good to know about all this. Thanks!

I haven't used `set` to make the script shorter. But I agree it's easy to miss.

About using `env`. Should I use that with `sh` as well or only for `bash` and more recent utilities? Or is Bourne Shell way too tightly coupled with this path that no system would dare move it?

Reply:
`/usr/bin/env`is the thing most linux distros respect. Bash can be in `/bin`, `/usr/bin`, or... Some users might have a more up to date one in `/usr/local/bin`. Or someone might be using nix with their fugly paths. Either way, we want to respect that.

`/bin/sh` is essentially a standard, so we can use it as is.

> Or is Bourne Shell way too tightly coupled with this path that no system would dare move it?

Not sure I get what you mean.

Reply:
> `/usr/bin/env` is the thing most linux distros respect.

Got it. I'm surely gonna start using it instead then. Thanks for the explanation!

> Not sure I get what you mean.

What I wanted to say is in that sense of being a standard, which no one would want to change to avoid (legacy) scripts from failing.